### PR TITLE
Enable tensorflow NLP examples using new API

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Type of Change

feature 
API not changed

## Description

detail description 
JIRA ticket: ILITV-2598

## Expected Behavior & Potential Risk

Enable tensorflow NLP examples using new API

## How has this PR been tested?

Pre-CI and models extension test.

## Dependency Change?

No.

### CODE REVIEW

**Comment by lvliang-intel** [comment]
extension test: https://inteltf-jenk.sh.intel.com/view/lpot-oob/job/intel-lpot-validation-top-extension-TFnewAPI/336/

**Review by zehao-intel — APPROVED** [review]

**Comment by lvliang-intel** [comment]
extension test:
https://inteltf-jenk.sh.intel.com/view/lpot-oob/job/intel-lpot-validation-top-extension-TFnewAPI/340/

File: examples/README.md [hunk 1]
```diff
@@ -216,13 +216,13 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>*BERT large SQuAD (Model Zoo)</td>
     <td>Natural Language Processing</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/nlp/bert_large_squad_model_zoo/quantization/ptq">pb</a></td>
+    <td><a href="./examples/tensorflow/nlp/bert_large_squad_model_zoo/quantization/ptq">pb</a></td>
```
Comment (chensuyue) [inline examples/README.md:219]: link need to fix

File: examples/tensorflow/nlp/bert_large_squad/quantization/ptq/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
+```
```
Comment (chensuyue) [inline examples/tensorflow/nlp/bert_large_squad/quantization/ptq/README.md:20]: > Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).

File: examples/tensorflow/nlp/bert_large_squad_model_zoo/quantization/ptq/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### Install Intel Tensorflow
+```python
+pip install intel-tensorflow
+```
```
Comment (chensuyue) [inline examples/tensorflow/nlp/bert_large_squad_model_zoo/quantization/ptq/README.md:20]: > Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).

**Pull request merged**